docs/quickstart.rst

```diff
@@ -152,7 +152,11 @@ In this chart, we see that the features ``temp`` and ``feelslike`` have a strong
 
 This visualizer plots a scatter diagram of the apparent temperature on the y axis and the actual measured temperature on the x axis and draws a line of best fit using a simple linear regression. Additionally, univariate distributions are shown as histograms above the x axis for temp and next to the y axis for feelslike.  The ``JointPlotVisualizer`` gives an at-a-glance view of the very strong positive correlation of the features, as well as the range and distribution of each feature. Note that the axes are normalized to the space between zero and one, a common technique in machine learning to reduce the impact of one feature over another.
 
-This plot is very interesting; first there appear to be some outliers in the dataset, where feelslike is approximately equal to 0.25. These instances may need to be manually removed in order to improve the quality of the final model because they could represent data input errors. Secondly, we can see that more extreme temperatures create an exaggerated effect in perceived temperature; the colder it is, the colder people are likely to believe it to be, and the warmer it is, the warmer it appears to be. Moderate temperatures feel like they do. This gives us the intuition that feelslike may be a better feature than temp, and if it is causing problems in our regression analysis, we should probably remove the temp variable in favor of feels like.
+This plot is very interesting because there appear to be some outliers in the dataset. These instances may need to be manually removed in order to improve the quality of the final model because they may represent data input errors, and potentially train the model on a skewed dataset which would return unreliable model predictions. The first instance of outliers occurs in the temp data where the `` feelslike``  value is approximately equal to 0.25 - showing a horizontal line of data, likely created by input error.
+
+We can also see that more extreme temperatures create an exaggerated effect in perceived temperature; the colder it is, the colder people are likely to believe it to be, and the warmer it is, the warmer it is perceived to be, with moderate temperatures generally having little effect on individual perception of comfort. This gives us a clue that `` feelslike``  may be a better feature than `` temp`` - promising a cleaner dataset, with less risk of running into outliers or errors.
+
+We can ultimately confirm the assumption by training our model on either value, and scoring the results. If the `` temp``  value is indeed less reliable, we should remove the `` temp``  variable in favor of `` feelslike`` . In the meantime, we will use the `` feelslike``  value due to the absence of outliers and input error.
```

Comment:
I think this is a very excellent conclusion from the visualization; thanks for adding it!